cherrypicks_test.go

```diff
@@ -398,7 +398,45 @@ func TestParseCherryTargetBranches(t *testing.T) {
 
 	for name, test := range tests {
 		t.Log(name)
-		res := parseCherryTargetBranches(test.body)
+		res, _ := parseCherryTargetBranches(test.body)
+		assert.Equal(t, test.expected, res[0])
+	}
+}
+
+func TestParseSingleLineCherryTargetBranches(t *testing.T) {
+	tests := map[string]struct {
+		body     string
+		expected string
+	}{
+		"master": {
+			body:     "cherry-pick to: `master`",
+			expected: "master",
+		},
+		"hosted": {
+			body:     "cherry-pick to: `hosted`",
+			expected: "hosted",
+		},
+		"staging": {
+			body:     "cherry-pick to: `staging`",
+			expected: "staging",
+		},
+		"feature-branch": {
+			body:     "cherry-pick to: `feature-independe_testing-1`",
+			expected: "feature-independe_testing-1",
+		},
+		"1.2.x": {
+			body:     "cherry-pick to: `1.2.x`",
+			expected: "1.2.x",
+		},
+		"1.2.x with escape char": {
+			body:     "cherry-pick to: `1.2.x`" + `\r`,
```

Comment:
Suggestion: 
Add one test with multiple branches on the same line :)

Reply:
I agree! The original test didn't do it so I thought it might not be necessary. Maybe do this for both parse-tests?